src/pcapi/routes/native/v1/serialization/offerers.py

```diff
@@ -22,3 +23,10 @@ class Config:
     withdrawalDetails: typing.Optional[str]
     address: typing.Optional[str]
     postalCode: typing.Optional[str]
+    venueTypeEnum: typing.Optional[str]
```

Comment:
```suggestion
    venueType: VenueType
```

et est-ce qu'on peut le rendre non-optionnel ? Par défaut ce serait "OTHER"